pkg/wallet/wallet.go

```diff
@@ -498,9 +498,25 @@ func (c *Wallet) Prove(authToken string, proofOptions *ProofOptions, credentials
 
 	presentation.Holder = proofOptions.Controller
 
-	err = c.addLinkedDataProof(authToken, presentation, proofOptions, purpose)
-	if err != nil {
-		return nil, fmt.Errorf("failed to prove credentials: %w", err)
+	switch proofOptions.ProofFormat {
+	case ExternalJWTProofFormat:
+		// TODO: look into passing audience identifier
```

Comment:
can you please create a task for this TODO in aries, so that we don't miss it.

Reply:
@sudeshrshetty good idea, done: https://github.com/hyperledger/aries-framework-go/issues/3354